jdk/make/CreateJars.gmk

```diff
@@ -271,6 +271,7 @@ $(IMAGES_OUTPUTDIR)/lib$(PROFILE)/_the.rt.jar.contents: $(IMAGES_OUTPUTDIR)/lib$
 	$(MKDIR) -p $(@D)
 	$(RM) $@ $@.tmp
 	$(GREP) -e '\.class$$' $(IMAGES_OUTPUTDIR)/lib$(PROFILE)/_the.jars.contents > $@.tmp
+	$(GREP) -e '\.properties$$' $(IMAGES_OUTPUTDIR)/lib$(PROFILE)/_the.jars.contents >> $@.tmp
```

Comment:
It doesn't seem correct to modify this file which OpenJDK also uses. Somehow the OpenJDK build puts .properties files into the rt.jar without this modification. For example jdk/src/share/classes/javax/sql/rowset/rowset.properties